### PR TITLE
Add customized network configuration for Azure installation

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1757229

### CODE REVIEW

**Comment by jboxman** [comment]
@kalexand-rh, it's possible to customize the network configuration for Azure, including opting to use OVN TP as the cluster network SDN.

Does the include order make sense?

Thanks.

File: installing/installing_aws/installing-aws-network-customizations.adoc [hunk 1]
```diff
@@ -60,6 +60,8 @@ include::modules/installation-aws-config-yaml.adoc[leveloffset=+2]
 include::modules/nw-modifying-operator-install-config.adoc[leveloffset=+1]
 include::modules/nw-operator-cr.adoc[leveloffset=+1]
 
+include::modules/installation-configure-proxy.adoc[leveloffset=+2]
```
Comment (kalexand-rh) [inline installing/installing_aws/installing-aws-network-customizations.adoc]: You have this file in different places in the AWS and Azure flows. I think I'd change this one.

File: installing/installing_azure/installing-azure-network-customizations.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+You must set most of the network configuration parameters during installation,
+and you can modify only `kubeProxy` configuration parameters in a running
+cluster.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+//* xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Configure an Azure account]
```
Comment (kalexand-rh) [inline installing/installing_azure/installing-azure-network-customizations.adoc]: Uncomment this xref.

File: modules/nw-operator-cr.adoc [hunk 3]
```diff
@@ -2,6 +2,7 @@
 //
 // * networking/cluster-network-operator.adoc
 // * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
 
 ifeval::["{context}" == "installing-aws-network-customizations"]
```
Comment (kalexand-rh) [inline modules/nw-operator-cr.adoc]: Do you need to update the if statements for Azure?

**Review by kalexand-rh — COMMENTED** [review]
I'd swap one of the include locations, but it overall LGTM. Please update the welcome page draft doc with the new URL.

File: modules/nw-operator-cr.adoc (hunk 3 above)
Comment (jboxman) [reply modules/nw-operator-cr.adoc]: Reworked conditionals.

**Comment by jboxman** [comment]
Verified on the bug.

**Pull request merged**